src/stores/RootStore.ts

```diff
@@ -73,9 +74,20 @@ export default class RootStore {
 		let messagesStoreState: MessagesStoreURLState = {};
 
 		if (activeWorkspace && isWorkspaceStore(activeWorkspace)) {
+			const clearEmpty = (filter: Partial<MessageFilterState> | Partial<EventsFilter>) => {
+				const tempFilter: Partial<MessageFilterState> | Partial<EventsFilter> = {};
+				Object.assign(tempFilter, filter);
```

Comment:
Also we can make some improvements by also deleting default values from url for example
`isSoftFilter` default value is false, so we can delete that
`flattenedListView` defaults to false

and also we can remove status filter from events if its value is `any`